example/android/app/src/main/java/com/example/MainApplication.java

```diff
@@ -1,42 +1,42 @@
 package com.example;
 
 import android.app.Application;
+import android.util.Log;
 
 import com.facebook.react.ReactApplication;
+import com.facebook.react.ReactInstanceManager;
 import com.facebook.react.ReactNativeHost;
 import com.facebook.react.ReactPackage;
 import com.facebook.react.shell.MainReactPackage;
 import com.facebook.soloader.SoLoader;
-import com.gettipsi.reactnativetwittersdk.TwitterReactPackage;
 
 import java.util.Arrays;
 import java.util.List;
 
 public class MainApplication extends Application implements ReactApplication {
 
-    private final ReactNativeHost mReactNativeHost = new ReactNativeHost(this) {
-        @Override
-        protected boolean getUseDeveloperSupport() {
-            return BuildConfig.DEBUG;
-        }
-
-        @Override
-        protected List<ReactPackage> getPackages() {
-            return Arrays.<ReactPackage>asList(
-                    new MainReactPackage(),
-                    new TwitterReactPackage()
```

Comment:
We should restore this lines, because without them, example application does not work. Just run `react-native link` in `example` folder.